### ISSUE TITLE
[Bug] MacOS MasterDetailPage MasterBehaviour.Popover doesn't hide the master page

### ISSUE THREAD

**Issue by kevcrooks** [xamarin/Xamarin.Forms#8984]
### Steps to Reproduce

1. Create a Xamarin.Mac project with Xamarin.Forms where the startup page is a MasterDetailPage
2. Set the Master and Detail pages to be empty with different coloured backgrounds
3. Set MasterBehavior = MasterBehavior.Popover and IsPresented = false:

```csharp
this.MainPage = new MasterDetailPage
{
    Master = myMaster,
    Detail = myDetail,
    MasterBehavior = MasterBehavior.Popover,
    IsPresented = false,
};
```

### Expected/Actual Behavior
The master page should not be shown, with "menu" or a hamburger shown. However the master page is always shown.

**Comment by charlesroddie** [comment]
This is a duplicate of https://github.com/xamarin/Xamarin.Forms/issues/3149 and is fixed in [4.5.0.187-pre2](https://docs.microsoft.com/en-us/xamarin/xamarin-forms/release-notes/4.5/4.5.0-pre2).

**Comment by jfversluis** [comment]
Taking into consideration the remaining lifetime of Xamarin.Forms, the platform this is about, the age of this issue and the traction this issue has seen I think it's unlikely that this will get fixed. If it is something that you still would like to have, please consider opening a PR. Thanks!
